content/en/docs/setup/install/multicluster/gateways/index.md

```diff
@@ -160,7 +160,7 @@ data:
 EOF
 {{< /text >}}
 
-For clusters that use CoreDNS:
+For clusters that use CoreDNS (< 1.4.0):
```

Comment:
Please use tabs for each option (kube-dns, CoreDNS 1.4.0 and earlier, and CoreDNS 1.4.0 and later. The way to use tabs is described here: https://istio.io/about/contribute/creating-and-editing-pages/#using-tabs

Reply:
Oh I fixed it, I actually meant `>= 1.4.0`.